package.json

```diff
@@ -24,15 +24,20 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "ejs": "1.0.x",
-    "express": "2.5.x",
-    "forever": "0.11.x",
-    "moment": "2.8.x",
+    "body-parser": "1.12.x",
+    "ejs": "2.3.x",
+    "errorhandler": "1.3.x",
+    "express": "4.12.x",
+    "express-dynamic-helpers-patch": "0.0.x",
```

Comment:
What do you think about using moment for ``timeago`` so we can remove the ``dynamicHelpers`` and ``express-dynamic-helpers-patch``? It shouldn't be hard, right? :)